src/test/java/bio/terra/integration/DataRepoClient.java

```diff
@@ -186,6 +186,15 @@ private String getLocationHeader(DataRepoResponse<JobModel> jobModelResponse) {
         return makeDrsRequest(path, HttpMethod.GET, entity, responseClass);
     }
 
+    public ResponseEntity<String> madeUnauthenticatedDrsRequest(String path,
```

Comment:
I think you want present test not past here.
```suggestion
    public ResponseEntity<String> makeUnauthenticatedDrsRequest(String path,
```